thoth/investigator/adviser_trigger/investigate_adviser_trigger.py

```diff
@@ -34,7 +34,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-@register_handler(AdviserTriggerMessage().topic_name, ["v1", "v2", "v3"])
+@register_handler(AdviserTriggerMessage().topic_name, ["v1", "v2", "v3", "4"])
```

Comment:
I wonder if we should keep the older versions there? I think the implementation raises KeyError for older messages as "authenticated" will not be present there.

Reply:
yes let use , `.get()` to keep it back compatible